packages/fx-core/src/plugins/resource/bot/languageStrategy.ts

```diff
@@ -101,7 +101,7 @@ export class LanguageStrategy {
     if (programmingLanguage === ProgrammingLanguage.JavaScript) {
       try {
         // fail to npm install @microsoft/teamsfx on azure web app, so pack it locally.
```

Comment:
Also remove the comment?

Reply:
Thanks for the comment, and I believe some comments should be added here since bot used remote build mode for deploy, and original meaning of this comment doesn't change.  I prefer to keep this.